src/handler.ts

```diff
@@ -2,7 +2,11 @@ import { BasicMouseHandler, DataGrid } from '@lumino/datagrid';
 import { Signal } from '@lumino/signaling';
 
 export default class RichMouseHandler extends BasicMouseHandler {
+  constructor() {
```

Comment:
I don't think you need to explicitly write a constructor

Reply:
You're right. I removed the constructor and fixed the package.json to pull from the right version of the tde-csvviewer module.